.github/workflows/ci.yml

```diff
@@ -56,8 +56,8 @@ jobs:
     if: ${{ github.event.pull_request.head.repo.full_name == '' || github.event.pull_request.head.repo.full_name == 'temporalio/sdk-go' }}
     runs-on: ubuntu-latest
     env:
-      SERVICE_ADDR: tinycicd.sdk.tmprl.cloud:7233
-      TEMPORAL_NAMESPACE: tinycicd.sdk
+      SERVICE_ADDR: sdk-ci.a2dd6.tmprl.cloud:7233
+      TEMPORAL_NAMESPACE: sdk-ci.a2dd6
       TEMPORAL_CLIENT_CERT: ${{ secrets.TEMPORAL_CLIENT_CERT }}
```

Comment:
I assume the new namespace has different certs?

Reply:
Yes. I already rotated the certs earlier today because the old ones had expired, and it was causing CI failures in the features repo.

Reply:
ah but it looks like the cloud tests are failing on this PR 

```
failed reaching server: last connection error: connection error: desc = "error reading server preface: remote error: tls: certificate required"
```

Reply:
Something I don't understand, though, is why the cloud runs are failing as if no cert had been provided at all:

https://github.com/temporalio/sdk-go/actions/runs/6151685113/job/16692274946?pr=1233
```
Run go test -v --count 1 -p 1 . -run "TestIntegrationSuite/TestBasic$"
  go test -v --count 1 -p 1 . -run "TestIntegrationSuite/TestBasic$"
  shell: /usr/bin/bash -e {0}
  env:
    SERVICE_ADDR: sdk-ci.a2dd6.tmprl.cloud:7233
    TEMPORAL_NAMESPACE: sdk-ci.a2dd6
    TEMPORAL_CLIENT_CERT: ***
    TEMPORAL_CLIENT_KEY: ***
=== RUN   TestIntegrationSuite
2023/09/11 20:57:08 Using custom client certificate
=== RUN   TestIntegrationSuite/TestBasic
=== NAME  TestIntegrationSuite
    integration_test.go:192: 
        	Error Trace:	/home/runner/work/sdk-go/sdk-go/test/integration_test.go:192
        	            				/home/runner/go/pkg/mod/github.com/stretchr/testify@v1.8.4/suite/suite.go:187
        	Error:      	Received unexpected error:
        	            	failed reaching server: last connection error: connection error: desc = "error reading server preface: remote error: tls: certificate required"
        	Test:       	TestIntegrationSuite
```

Still looking into why this is happening.

Reply:
D'oh. There were repo-local certs that overrode the org-wide certs that I rotated. CI should be passing now.